include/spdlog/details/pattern_formatter_impl.h

```diff
@@ -483,7 +496,7 @@ class full_formatter SPDLOG_FINAL : public flag_formatter
         msg.color_range_start = msg.formatted.size();
         msg.formatted << level::to_str(msg.level);
         msg.color_range_end = msg.formatted.size();
-        msg.formatted << "] " << fmt::StringRef(msg.raw.data(), msg.raw.size());
+        msg.formatted << "] " << fmt::string_view(msg.raw.data(), msg.raw.size());
```

Comment:
It is very much possible we could be using this throughout the codebase rather than `std::string &` or similar for added benefit.